docs/configuration/identity-providers/oidc.md

```diff
@@ -138,39 +140,43 @@ identity_providers:
 ## Options
 
 ### hmac_secret
+
 <div markdown="1">
 type: string
 {: .label .label-config .label-purple } 
 required: yes
 {: .label .label-config .label-red }
 </div>
 
-The HMAC secret used to sign the [OpenID Connect] JWT's. The provided string is hashed to a SHA256 byte string for
-the purpose of meeting the required format.
-
-Can also be defined using a [secret](../secrets.md) which is the recommended for containerized deployments.
+The HMAC secret used to sign the [OpenID Connect] JWT's. The provided string is hashed to a SHA256
+byte string for the purpose of meeting the required format. It can also be defined using a
+[secret](../secrets.md) which is the recommended for containerized deployments. You must
```

Comment:
I think it makes sense to have the information about this being definable as a secret by itself, it draws more attention to it. Same for the issuer private key (and they should be a copy paste). Thoughts?

Reply:
I put the note about the secrets on its own line to make the reader more aware. I also swapped "Can" with "Should".